installer/dual_rootfs_device.go

```diff
@@ -28,6 +28,14 @@ import (
 	log "github.com/sirupsen/logrus"
 )
 
+const (
+	verifyRebootError = "Reboot to new update failed. " +
+		"Expected \"upgrade_available\" flag to be true but it was false. " +
+		"Either the switch to the new partition was unsuccessful, or the bootloader rolled back"
```

Comment:
there is a full-stop ('.') at the end of `verifyRebootError` and there is none at the end of `verifyRollbackRebootError` I remember go once told me not to use '.' in error messages, but take your pick. I guess it would be good to have it uniform.